Packet++/src/TextBasedProtocol.cpp

```diff
@@ -127,6 +128,11 @@ void TextBasedProtocolMessage::parseFields()
 			delete newField;
 			break;
 		}
+		if (iterations == 100)
+    { 
```

Comment:
can you please use tabs instead of spaces to align with the rest of the file?

Reply:
should be changed